bokehjs/test/document.coffee

```diff
@@ -550,7 +550,7 @@ describe "Document", ->
     expect(out).to.be.equal "Bokeh: JS/Python version mismatch\nBokeh: Library versions: JS (#{js_version})  /  Python (0.0.1)\n"
 
     parsed['version'] = "#{js_version}-foo"
-    out = stdoutTrap -> Document.from_json_string(JSON.stringify(parsed))
+    out = stderrTrap -> Document.from_json_string(JSON.stringify(parsed))
     expect(out).to.be.equal ""
```

Comment:
OK but let's also add a test for the other three cases that shouldn't skip, and ensure that they do print warnings when they are supposed to, and don't when they aren't.